### PR TITLE
Add the slices.py helper tool

### PR DESCRIPTION
**Pull request by withrocks**

### CODE REVIEW

File: clarity-ext-scripts/deployment/manifest.txt [hunk 1]
```diff
@@ -0,0 +1,16 @@
+ConfigSlicerVersion=3.1.14.7
```
Comment (johandahlberg) [inline clarity-ext-scripts/deployment/manifest.txt:1]: Should this one be commited?

File: clarity-ext-scripts/README.md [hunk 2]
```diff
@@ -16,3 +18,51 @@ branch `covid` on:
...
+This configuration file should contain information on this format:
+
+    staging:
+      server: server 
+      username: username
+      password: password 
+    prod:
+      server: server 
+      username: username
+      password: password
```
Comment (johandahlberg) [inline clarity-ext-scripts/README.md:47]: Good idea!

File: clarity-ext-scripts/deployment/get-slice-tool.sh [hunk 3]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+# Usage: ./get-slice-tool <SERVERNAME>
```
Comment (johandahlberg) [inline clarity-ext-scripts/deployment/get-slice-tool.sh:3]: I guess you actually need both user and server name here?

File: clarity-ext-scripts/deployment/manifest.txt (hunk 1 above)
Comment (withrocks) [reply clarity-ext-scripts/deployment/manifest.txt:1]: Yeah, I was thinking that. It's like a lock file which says what we want to export. Then we don't have to do this selection step again and again, but we might need to export the package again (similar to specifying requirements in requirements.txt and actually pip installing, we check in the requirements file but not the libraries).

File: clarity-ext-scripts/deployment/get-slice-tool.sh (hunk 3 above)
Comment (withrocks) [reply clarity-ext-scripts/deployment/get-slice-tool.sh:3]: Right, unless the username is the same.

Comment (withrocks) [reply clarity-ext-scripts/deployment/get-slice-tool.sh]: Fixed.

**Pull request merged**